src/nvim/os/win_defs.h

```diff
@@ -6,16 +6,22 @@
 #define TEMP_DIR_NAMES {"$TMP", "$TEMP", "$USERPROFILE", ""}
 #define TEMP_FILE_PATH_MAXLEN _MAX_PATH
 
-// Defines needed to fix the build on Windows:
-// - DFLT_DIR
-// - DFLT_BDIR
-// - DFLT_VDIR
-// - EXRC_FILE
-// - VIMRC_FILE
-// - SYNTAX_FNAME
-// - DFLT_HELPFILE
-// - SYS_VIMRC_FILE
-// - SPECIAL_WILDCHAR
+// Windows system-dependent file names
+#ifndef SYS_VIMRC_FILE
+# define SYS_VIMRC_FILE "$VIM\\sysinit.vim"
+#endif
+#ifndef DFLT_HELPFILE
+# define DFLT_HELPFILE  "$VIMRUMTIME\\doc\\help.txt"
+#endif
+#ifndef SYNTAX_FNAME
+# define SYNTAX_FNAME   "$VIMRUNTIME\\syntax\\%s.vim"
```

Comment:
Ok.

@justinmk I noticed that `os_defs.h` has the following:

* `FILETYPE_FILE`
* `FTPLUGIN_FILE`
* `INDENT_FILE`
* `FTOFF_FILE`
* `FTPLUGOF_FILE`
* `INDOFF_FILE`
* `DFLT_ERRORFILE`

Should those also be moved to `globals.h` too since they are defined the same on all platforms? I think it would be weird to add `EXRC_FILE`, `VIMRC_FILE`, etc. in one place (`globals.h`) and have the other ones in `os_defs.h`.

Thoughts?